src/did/methods/ion.js

```diff
@@ -34,5 +33,5 @@ export {
   create,
   sign,
   verify,
-  resolve
+  resolve,
```

Comment:
👍🏽 can we add a linting rule for this in [eslintrc.cjs](https://github.com/TBD54566975/web5-js/blob/main/.eslintrc.cjs)?